contrib/openvpn/files/openvpn-server

```diff
@@ -0,0 +1,7 @@
+# openvpn server service
+
+type = process
+command = /usr/bin/openvpn --status /var/log/openvpn-server-status.log --status-version 2 --suppress-timestamps --log /var/log/openvpn-server-service.log --config ${OPENVPN_SERVER:-default}.conf
```

Comment:
${} is substituted from env, but there is no way to set an env for a service. you'd probably want env-file= too with that (maybe /etc/default/openssl-{client,server}) and set OPENVPN_{CLIENT,SERVER}_CONFIG=/etc/openvpn/.... in it

Reply:
also, is the server-status useful for anything, and why version =2 (there's 3 too, defaults to 1, etc)

Reply:
> there is no way to set an env for a service

Does `dinitctl setenv` not work for variable substitution? Or is it a bad idea to rely on it?

Also, the sample services for systemd used `%i` as a method of selecting a VPN tunnel, but I didn't see a convenient analogue for this in dinit. If you think an `env-file` is the best way, then so shall it be. 

> is the server-status useful for anything

Admittedly I have no idea. I've never administrated an OpenVPN server. 

> why version =2

The example service files provided by OpenVPN use version 2, and I assume they know more than I do.

Reply:
> Does dinitctl setenv not work for variable substitution? Or is it a bad idea to rely on it?

that sets it for every service which isn't what you want

>  If you think an env-file is the best way, then so shall it be.

i just don't see any other way really